cli_reference/openshift_developer_cli/understanding-odo.adoc

```diff
@@ -30,11 +30,11 @@ Application::
 An application is a program designed for end users. An application consists of multiple microservices or components that work individually to build the entire application.
 Examples of applications: a video game, a media player, a web browser.
 Component::
-A component is a building block of an application. Typically it is a software package, a web service, a web resource, or a module that encapsulates a set of related functions or data. 
+A component is a set of kubernetes resources which host code or data. These resources run in a specific order for the overall software system to work. Each component can be run and deployed separately.
```

Comment:
My understanding from Gorkem's definition is that the resources need to be running in order for the overall system to work. The definition doesn't seem to imply that these resources should be running in a specific order as mentioned here.

Besides, as a dev, I can say that we do start things in a certain order when a component is pushed to OpenShift cluster. But beyond that, the resources don't need to be running in a specific order for things to work.

Reply:
```suggestion
A component is a set of kubernetes resources which host code or data.  Each component can be run and deployed separately.
```
"These resources run in a specific order for the overall software system to work." This sentence can be a little bit misleading. I would just remove it.